### DISCUSSION TITLE
Expain differences between items and allItems

### DISCUSSION THREAD

**Discussion by manuelmeister** [ecamp/hal-json-vuex#283] [Q&A] [answered]
When should I use items and when allItems?

**Comment by carlobeltrame** [comment, marked as answer]
`allItems` includes items which are marked as `deleting`. Normally you should use `items`, to benefit from the optimistic deleting behaviour (i.e. the deleted item already vanishes from `items` while the DELETE request is still ongoing).

https://github.com/ecamp/hal-json-vuex/blob/e4d893b7427595fe907e83f993ffe911c9cb8384/src/CanHaveItems.js#L20:L21
